cedar-policy-cli/Cargo.toml

```diff
@@ -28,6 +28,7 @@ partial-validate = ["cedar-policy/partial-validate"]
 assert_cmd = "2.0"
 tempfile = "3"
 glob = "0.3.1"
+predicates = "3.1.0"
```

Comment:
See https://github.com/cedar-policy/cedar/pull/798#discussion_r1569603483 for rationale